lib/matplotlib/axes/_axes.py

```diff
@@ -6858,42 +6858,39 @@ def hist2d(self, x, y, bins=10, range=None, density=False, weights=None,
 
             The bin specification:
 
-                - If int, the number of bins for the two dimensions
-                  (nx=ny=bins).
-
-                - If ``[int, int]``, the number of bins in each dimension
-                  (nx, ny = bins).
-
-                - If array_like, the bin edges for the two dimensions
-                  (x_edges=y_edges=bins).
-
-                - If ``[array, array]``, the bin edges in each dimension
-                  (x_edges, y_edges = bins).
+            - If int, the number of bins for the two dimensions
+              (nx=ny=bins).
+            - If ``[int, int]``, the number of bins in each dimension
+              (nx, ny = bins).
+            - If array_like, the bin edges for the two dimensions
+              (x_edges=y_edges=bins).
+            - If ``[array, array]``, the bin edges in each dimension
+              (x_edges, y_edges = bins).
 
             The default value is 10.
 
         range : array_like shape(2, 2), optional, default: None
-             The leftmost and rightmost edges of the bins along each dimension
-             (if not specified explicitly in the bins parameters): ``[[xmin,
-             xmax], [ymin, ymax]]``. All values outside of this range will be
-             considered outliers and not tallied in the histogram.
+            The leftmost and rightmost edges of the bins along each dimension
+            (if not specified explicitly in the bins parameters): ``[[xmin,
+            xmax], [ymin, ymax]]``. All values outside of this range will be
+            considered outliers and not tallied in the histogram.
 
         density : bool, optional, default: False
-             Normalize histogram.  *normed* is a deprecated synonym for this
-             parameter.
+            Normalize histogram.  *normed* is a deprecated synonym for this
+            parameter.
 
         weights : array_like, shape (n, ), optional, default: None
             An array of values w_i weighing each sample (x_i, y_i).
 
         cmin : scalar, optional, default: None
-             All bins that has count less than cmin will not be displayed and
-             these count values in the return value count histogram will also
-             be set to nan upon return
+            All bins that has count less than cmin will not be displayed (set
+            to none before passing to imshow) and these count values in the
```

Comment:
none is a bit unspecific. Would use NaN or None. (Is there any reason we use `None` instead of `np.nan` in the code?)

Reply:
replaced docstring by NaN.
dunno about the implementation, but assigning None to a float array (which histogram2d always returns) has the same effect as assigning nan anyways...